cypress/models/loginPage.js

```diff
@@ -1,14 +1,16 @@
 import LoginPage from '../page-objects/login.page';
+import UniversalPages from '../page-objects/universal.pages';
 import {FAKE_DATA} from "../testData/testData";
 
+const universalPages = new UniversalPages();
 const loginPage = new LoginPage();
 
 export function fillInTheRegistrationFormAndClickTheSendButton() {
     loginPage.userNameField().type(FAKE_DATA.name).should("be.visible");
     loginPage.userEmailField().type(FAKE_DATA.email).should("be.visible");
     loginPage.currentAddress(0).type(FAKE_DATA.currentAddress).should("be.visible");
     loginPage.permanentAddress(0).type(FAKE_DATA.permanentAddress).should("be.visible");
-    loginPage.submitBtn().should("be.visible").click();
+    universalPages.submitBtn().should("be.visible").click();
 }
 
 export function checkEmailInResult(expectedEmail) {
```

Comment:
There is no verification, from 14 row to 28
cy. contains() and  .should('contain', 'text') it's not the same!

Reply:
Done